browser/brave_ads/tooltips/BUILD.gn

```diff
@@ -24,4 +25,8 @@ source_set("tooltips") {
     "//chrome/browser/profiles:profile",
     "//ui/base",
   ]
+
+  if (toolkit_views) {
```

Comment:
@emerick This deps including should be deleted from `//brave/browser/brave_ads/tooltips` target.
Instead, let's add this `//brave/browser/ui/views/brave_tooltips` target to `//brave/browser/ui`

Reply:
Fixed.